GDJS/Runtime/runtimeobject.ts

```diff
@@ -940,6 +940,47 @@ namespace gdjs {
       array.removeAtIndex(index);
     };
 
+    /**
+     * Shortcut to return the size of an array or structure variable.
+     * @returns {number}
+     */
+    getVariableSize = function (array: gdjs.Variable): number {
+      return array.getChildrenCount();
+    };
+
+    /**
+     * Shortcut to get the value of an array variable at a given index.
+     * @returns {string | number | boolean}
+     */
+    getVariableValueAtIndex = function (array: gdjs.Variable, index: number) {
```

Comment:
Here this is `getVariableValueAtIndex` but for scene/global variables it's called `getVariableValueAt`.

We should choose one and stick with it.